### DISCUSSION TITLE
Editing page always load

### DISCUSSION THREAD

**Discussion by GvcZhang** [GitbookIO/community#281] [0. General bugs and questions] [answered]
![image](https://github.com/GitbookIO/community/assets/45255699/e0135dbe-a2e9-4c8c-a2f8-b7767b2b6ca0)
it often into a load stage when I editing content. 

why?

**Comment by aikoii** [comment]
Sorry about this, @GvcZhang! Could you email us at [support@gitbook.com](mailto:support@gitbook.com) and point me to space or change request where you have those issues?

**Reply by GvcZhang** [reply]
Sorry, i can not understand what your means. Do you want me describe the detail of this issue in an email?

**Reply by aikoii** [reply]
Hey @GvcZhang , sorry for not making it clearer. Yes! It would be great if you could you send me the link (in-app) where you see this happening, it would be helpful if you could share a short video too 🙏

**Comment by GvcZhang** [comment]
[app.gitbook.com-1690536689861.log](https://github.com/GitbookIO/community/files/12235406/app.gitbook.com-1690536689861.log)

Hi @aikoii, i am not sure if the problem of network, i use a proxy to connect Gitbook. And the above log file is exported from console of Chrome DevTool when the editing page is reload. I hope it's useful.

**Reply by GvcZhang** [reply]
Also, i can record a video when page reloading if you need.

**Reply by aikoii** [reply, marked as answer]
Hey @GvcZhang, very sorry for the late reply 🙏! Yes, it seems related to your network. You can check our [firewall safelist](https://docs.gitbook.com/faq/connectivity-issues) page for tips on accessing GitBook. VPN could help, but might not work in all cases.
